### ISSUE TITLE
Mullvad Browser depends on dbus-glib on Linux

### ISSUE THREAD

**Issue by vecna13** [mullvad/mullvad-browser#117]
If dbus-glib is not installed, running ./start-mullvad-browser.desktop silently fails.

On Debian:

```
sudo apt remove libdbus-glib-1-2
./start-mullvad-browser.desktop
```

On Fedora:

```
sudo dnf remove dbus-glib
./start-mullvad-browser.desktop
```

Tested with mullvad-browser-linux64-12.5.1_ALL on Debian 11, Fedora 37, and Fedora 38, all as VMs in Qubes.

I don't know if the answer is to bundle dbus-glib with Mullvad Browser or just include a note if it's unable to start, but as a user, it is confusing for the browser to just fail to launch. This scenario may come up for instance if the user has Firefox installed (and dbus-glib as a dependency), downloads Mullvad Browser (which runs at this time), decides it's the only browser they need, then uninstalls Firefox so the package manager no longer thinks it needs dbus-glib.

**Comment by vecna13** [comment]
This appears to be an upstream issue. It would be nice if the launching script included a warning if there are missing dependencies, though.

I found a related Tor Browser issue: [tor browser relies on external glib/dbus installation](https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/10014)

**Comment by vecna13** [comment]
Same issue filed for Tor Browser: [Tor Browser silently fails if dbus-glib is not installed on Linux](https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/41914)

**Comment by c3Ls1US** [comment]
> Same issue filed for Tor Browser: [Tor Browser silently fails if dbus-glib is not installed on Linux](https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/41914)

This issue has been around for years, and, more importantly, [this library is deprecated at this point](https://github.com/d-bus/dbus-glib). Apparently, the team won't fix this until Firefox 128:

> Yep, but Firefox 120+ and it isn't trivial to backport.
I propose to close this as a wontfix and wait for 128.

**Comment by ruihildt** [comment]
This has now been fixed with the Mullvad Browser 14.0 release.

See: https://github.com/mullvad/mullvad-browser/releases/tag/14.0
